x-pack/plugins/search_indices/public/hooks/api/use_index.ts

```diff
@@ -18,7 +18,7 @@ export const useIndex = (indexName: string) => {
     refetchInterval: POLLING_INTERVAL,
     refetchIntervalInBackground: true,
     refetchOnWindowFocus: 'always',
-    retry: true,
+    retry: false,
```

Comment:
When retry is `true`, the api try to refetch even if there is a fetch error. So there is no way to confirm if fetchIndex had a failure since `isLoading` & `isSuccess` will be false. 
 Setting retry to `false`, would stop refetch when last fetch failed and will save the error which can help to show error message in front end.

Reply:
updated in 13ac078b449f62ce7f9d62a26e447485949134e6